File: video/v1/recording/read-default/read-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Video.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var recordings = RecordingResource.Read();

        foreach(var record in recordings) {
           Console.WriteLine(record.AccountSid);
        }
    }
}

File: api/v2010/add_on_result/read-default/read-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

add_on_results = @client.recordings('REaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                        .add_on_results
                        .list

add_on_results.each do |record|
  puts record.sid
end
